### DISCUSSION TITLE
Change radio station from menu

### DISCUSSION THREAD

**Discussion by viewsourc3** [jacklandrin/OnlySwitch#75] [Ideas]
I was wondering if it is doable to add the capability to change the radio stations from the menu itself. 

The user would add the stations form the configuration window but could change the stations from the drop down menu.

**Comment by jacklandrin** [comment]
@viewsourc3 You can directly change stations by media keys(next, previous, pause/play) on the build-in keyboard, or click control center to change the next station.

**Reply by viewsourc3** [reply]
so cool, thank you!
